addons/mail/static/src/xml/activity.xml

```diff
@@ -1,45 +1,86 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
     <t t-name="mail.activity_items">
-        <div t-if="activities.length" class="o_thread_date_separator o_border_dashed">
-            <span class="o_thread_date">Planned</span>
+        <div t-if="activities.length" class="o_thread_date_separator o_border_dashed" data-toggle="collapse" data-target="#o_chatter_planned_activities">
+            <t t-set="today_activities" t-value="0"/>
+            <t t-set="planned_activities" t-value="0"/>
+            <t t-set="overdue_activities" t-value="0"/>
+            <t t-foreach="activities" t-as="a">
+                <t t-if="a.state == 'today'" t-set="today_activities" t-value="today_activities + 1"/>
+                <t t-if="a.state == 'overdue'" t-set="overdue_activities" t-value="overdue_activities + 1"/>
+                <t t-if="a.state == 'planned'" t-set="planned_activities" t-value="planned_activities + 1"/>
```

Comment:
It would be better if variables `nbTodayActivities`, etc was computed in javascript instead ;)